pkg/adsc/delta.go

```diff
@@ -385,6 +430,102 @@ func initWatch(typeURL string, resourceName string) Option {
 	}
 }
 
+// DeltaMcpHandler is a function that will be called when a delta MCP config resource is received.
+// The entity is converted from a original `mcp.Resource`.
+type DeltaMcpConfigHandler func(gvk config.GroupVersionKind, namespaced, name string, entity *config.Config)
```

Comment:
```suggestion
type DeltaMcpConfigHandler func(gvk config.GroupVersionKind, entity *config.Config)
```

Isn't ns/name contained in config?